actionpack/lib/action_dispatch/routing/polymorphic_routes.rb

```diff
@@ -79,7 +79,7 @@ module PolymorphicRoutes
       #   polymorphic_url([blog, post], anchor: 'my_anchor', script_name: "/my_app")
       #     # => "http://example.com/my_app/blogs/1/posts/1#my_anchor"
       #
-      # For all of these options, see the documentation for <tt>url_for</tt>.
+      # For all of these options, see the documentation for {url_for}[rdoc-ref:Routing::UrlFor].
```

Comment:
Yeah, I believe so. The link worked for me, but I'm not entirely sure on the documentation guidelines for reference links. Should I update it with the `ActionDispatch::`?